app/controllers/editions_controller.rb

```diff
@@ -116,6 +119,20 @@ def update_tagging
     }
   end
 
+  def update_related_external_links
+    artefact = resource.artefact
+    external_links = params.require(:artefact).permit(external_links_attributes: [:title, :url, :id, :_destroy])
+
+    artefact.external_links_attributes = external_links["external_links_attributes"]
+
+    if artefact.save
+      flash[:success] = "External links have been saved."
+    else
+      flash[:danger] = "Not saved. Links should begin with 'http'."
```

Comment:
Gave a look into this and right now, independently from the error, the error message would be something like "Artefact external link is invalid". 

I could try to return a customised error here:
https://github.com/alphagov/govuk_content_models/blob/master/app/models/artefact_external_link.rb

What do you suggest?